lib/housing-wanted/housing-wanted.controllers.js

```diff
@@ -1,7 +1,133 @@
 module.exports = (models) => {
     const Boom = require('boom');
-  
-    return {}
-  
-  };
-  
\ No newline at end of file
+    const Sequelize = require('sequelize');
+    const Op = Sequelize.Op;
+    const housingWantedModels = require('./housing-wanted.models');
+    const accountLib = require('../account/account.controllers')(models).lib;
+    const locationLib = require('../location/location.controllers')(models).lib;
+    const Log = require('../log/log.controllers')(models).lib;
+
+    return {
+      postHousingWanted: async function(request, h){
+        /*
+          # Require Account for submitting fields other than address
+        */
+        let payload = request.payload;
+        let account;
+        try {
+         account = await accountLib.getAccount(request.auth.credentials);
+        } catch (e) {
+         throw Boom.badImplementation('Error during getAccount(request.auth.credentials). ' + e);
```

Comment:
this should be changed to `throw Boom.badImplementation('Error during getAccount(request.auth.credentials). ', e);`

Reply:
Yes! Thank you! Done, so I'm resolving this conversation.